packages/runtime/src/widgets/Dropdown.tsx

```diff
@@ -39,6 +64,43 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
     },
     [action],
   );
+
+  const { namedData } = useTemplateData({
+    data: itemTemplate?.data,
+    name: itemTemplate?.name,
+  });
+
+  const options = useMemo(() => {
+    if (values?.items) {
```

Comment:
Can you confirm what the behavior is for flutter? What is the order of preference or perhaps they concatenate the options if both exist?

Reply:
It concatenate both, items and item-template if available, and priority is given to items. so as per that I updated the PR